tests/test_schemas.py

```diff
@@ -0,0 +1,172 @@
+            "@type": "rdfs:Class",
+            "rdfs:comment": "This is a dummy test class",
+            "rdfs:label": "Test",
+            "rdfs:subClassOf": [{"@id": "bts:Thing"}],
+            "schema:isPartOf": {"@id": "http://schema.biothings.io"},
+            "schema:rangeIncludes": [{"@id": "bts:TestStart"}, {"@id": "bts:TestEnd"}],
+            "sms:displayName": "Test",
+            "sms:required": "sms:true",
+            "sms:requiresComponent": [
+                {"@id": "bts:Test_Comp_1"},
+                {"@id": "bts:Test_Comp_2"},
+            ],
+            "sms:requiresDependency": [
+                {"@id": "bts:Test_Dep_1"},
+                {"@id": "bts:Test_Dep_2"},
+            ],
+            "sms:validationRules": ["Rule_1", "Rule_2"],
+        }
+
+        assert expected == actual
+
+    def test_get_property(self, helpers):
+
+        se_obj = helpers.get_schema_explorer("example.model.jsonld")
+
+        actual = df_parser.get_property(
+            se=se_obj,
+            property_display_name="Test",
+            property_class_name="Prop_Class",
+            description="This is a dummy test property",
+            requires_range=["Test_Start", "Test_End"],
+            requires_dependencies=["Test_Dep_1", "Test_Dep_2"],
+            required=True,
+            validation_rules=["Rule_1", "Rule_2"],
+        )
+
+        expected = {
+            "@id": "bts:test",
+            "@type": "rdf:Property",
+            "rdfs:comment": "This is a dummy test property",
+            "rdfs:label": "test",
+            "schema:isPartOf": {"@id": "http://schema.biothings.io"},
+            "schema:rangeIncludes": [{"@id": "bts:TestStart"}, {"@id": "bts:TestEnd"}],
+            "sms:displayName": "Test",
+            "sms:required": "sms:true",
+            "schema:domainIncludes": {"@id": "bts:PropClass"},
+            "sms:requiresDependency": [
+                {"@id": "bts:Test_Dep_1"},
+                {"@id": "bts:Test_Dep_2"},
+            ],
+            "sms:validationRules": ["Rule_1", "Rule_2"],
+        }
+
+        assert expected == actual
+
+    def test_attribute_exists(self, helpers):
+
+        se_obj = helpers.get_schema_explorer("example.model.jsonld")
+
+        # test when attribute is present in data model
+        attribute_present = df_parser.attribute_exists(se_obj, "Patient")
+
+        # test when attribute is not present in data model
+        attribute_absent = df_parser.attribute_exists(se_obj, "RandomAttribute")
+
+        assert attribute_present == True
+        assert attribute_absent == False
+
+    def test_check_schema_definition(self, helpers):
+
+        data_model_csv_path = helpers.get_data_path("example.model.csv")
+
+        example_model_df = pd.read_csv(data_model_csv_path)
+
+        # when all required headers are provided in the CSV data model
+        actual_df = df_parser.check_schema_definition(example_model_df)
+
+        assert actual_df is None
```

Comment:
Good catch noticing the return type discrepancy in `check_schema_definition` @BrunoGrandePhD. Per your comment, you're saying we should return `bool` values rather than returning `None`?

I'll address this in a separate PR once we've decided if we want to stick to `bool` or change the type hints.